proton_driver/util/compat.py

```diff
@@ -5,6 +5,15 @@
 except ImportError:
     import dummy_threading as threading  # noqa: F401
 
+import json  # noqa: F401
+try:
+    import orjson as json  # noqa: F811
+except ImportError:
+    try:
+        import ujson as json  # noqa: F811,F401
+    except ImportError:
+        pass
```

Comment:
move 'import json' to line 15. I am still wonder why try to import orjson/ujson instead of just use standard json lib.

Reply:
orjson/ujson have higher performance than standard json lib.